packages/toolpad-app/src/toolpad/Home.tsx

```diff
@@ -463,33 +464,32 @@ function AppsListView({
   error,
   setDeletedApp,
 }: AppViewProps) {
+  if (status === 'success' && apps.length <= 0) {
+    return <React.Fragment>No apps yet</React.Fragment>;
+  }
+
+  if (status === 'error') {
+    return <ErrorAlert error={error} />;
+  }
+
   return (
     <Table aria-label="apps list" size="medium">
       <TableBody>
-        {(() => {
-          switch (status) {
-            case 'loading':
-              return <AppRow />;
-            case 'error':
-              return <Alert severity="error">{(error as Error)?.message}</Alert>;
-            case 'success':
-              return apps.length > 0
-                ? apps.map((app) => {
-                    const activeDeployment = activeDeploymentsByApp?.[app.id];
-                    return (
-                      <AppRow
-                        key={app.id}
-                        app={app}
-                        activeDeployment={activeDeployment}
-                        onDelete={() => setDeletedApp(app)}
-                      />
-                    );
-                  })
-                : 'No apps yet';
-            default:
-              return '';
-          }
-        })()}
+        {status === 'loading' ? (
```

Comment:
just nitpicking but maybe it's slightly easier to understand if this loading check is done above like the error check, and then here we use the ternary operator to check if there are no apps or more?

Reply:
I refactored it a bit for readability